### PR TITLE
Add unit tests for NewBookmarksDao.kt

### PR DESCRIPTION
**Pull request by bhaktideshmukh**
Fixes #

**Screenshots**

### CODE REVIEW

**Comment by kelson42** [comment]
@bhaktideshmukh what is the status of this Pr? Which tivket is fixed?

**Comment by gouri-panda** [comment]
>Which ticket is fixed? Any idea why the ci fails?

@kelson42 after #2668 it should be fine hopefully.

@bhaktideshmukh  In order to create a test(generally)  PR, you have to create an issue. See #1827

**Comment by bhaktideshmukh** [comment]
fixes #1827

**Comment by bhaktideshmukh** [comment]
:app:jacocoInstrumentationTestReport , I don't think this error is related to my code. Do you how to fix it?

**Comment by bhaktideshmukh** [comment]
:app:jacocoInstrumentationTestReport , I don't think this error is related to my code. Do you know how to fix it?

**Comment by gouri-panda** [comment]
@bhaktideshmukh You are good to go!

File: core/src/test/java/org/kiwix/kiwixmobile/core/dao/NewBookmarksDaoTest.kt [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+      )
+    } returns queryBuilder
+    every { queryBuilder.order(BookmarkEntity_.bookmarkTitle) } returns queryBuilder
+    val query: Query<BookmarkEntity> = mockk(relaxed = true)
+    every { queryBuilder.build() } returns query
+    every { bookmarkItem.zimId } returns ""
+    every { bookmarkItem.zimName } returns ""
+    every { bookmarkItem.databaseId } returns 0L
+    newBookmarksDao.getCurrentZimBookmarksUrl(zimFileReader)
+    val bookmarkEntity: BookmarkEntity = mockk()
```
Comment (gouri-panda) [inline core/src/test/java/org/kiwix/kiwixmobile/core/dao/NewBookmarksDaoTest.kt:71]: nit: `bookmarkEntity` is never used. You can delete it.

**Review by gouri-panda — APPROVED** [review]

**Comment by gouri-panda** [comment]
@bhaktideshmukh You should create a separate issue for this PR in order to merge this PR.

**Comment by kelson42** [comment]
@gouri-panda I don't understand this PR is good to merge? Or not?

**Comment by gouri-panda** [comment]
@kelson42 Generally it's a good idea to create a separate issue for PR.

**Comment by kelson42** [comment]
I'll merge this PR, the role of this PR is quite clear and a ticket would not add that much more info IMO.

**Comment by bhaktideshmukh** [comment]
@gouri-panda @kelson42 Thanks for the review

**Pull request merged**